explainers/subresource-loading.md

```diff
@@ -0,0 +1,102 @@
+# Explainer: Subresource loading with Web Bundles
+
+Last updated: March 2020
+
+We propose a new approach to load a large number of resources efficiently using
+a format that allows multiple resources be be bundled, e.g.
+[Web Bundles](https://web.dev/web-bundles/).
+
+## Backgrounds
+
+- Loading many unbundled resources is still slower in 2020. We concluded that
+  [bundling was necessary in 2018](https://v8.dev/features/modules#bundle), and
+  our latest local measurement still suggests that.
+
+- Caching stories with JS bundlers (e.g. webpack) is tough. They are pretty good
+  tools but configuring them to work in an optimal way is tough, and sometimes
+  also conflicts with new requirements like dynamic bundling (e.g. small edit
+  with tree shaking could invalidate everything).
+
+- With JS bundlers, parsing and execution needs to wait for the full bytes to
```

Comment:
I think this is not true for parsing.

Chrome is already doing streaming parsing / compiling for classic scripts (probably scaling it to more than one processor is hard, though), and for module scripts each module is parsed / compiled as soon as it is downloaded, without waiting for the entire module tree.

Reply:
Would just dropping 'parsing and' make sense?

Reply:
Thanks. Let's drop the parsing.